source/manifest.json

```diff
@@ -17,23 +18,31 @@
 	},
 	"permissions": [
 		"storage",
+		"scripting",
 		"contextMenus",
 		"activeTab",
```

Comment:
Why is this permission required? There is a side-effect with Firefox MV3: this extension now "needs attention" on every tab that isn't GitHub.

<img width="66" alt="Screenshot 2024-06-24 at 12 26 19" src="https://github.com/refined-github/refined-github/assets/217628/a22b7c6d-a3c4-48a9-b73e-73c556847980">

Reply:
That's not it, see #7477 

The permission is required in MV3 to add support for GitHub Enterprise